src/test/javascript/cypress/support/authentication.ts

```diff
@@ -0,0 +1,42 @@
+Cypress.Commands.add('loginUsing', (username: string, password: string) => {
+  cy.server();
```

Comment:
This is nice, also check out [this](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=video&cd=&cad=rja&uact=8&ved=2ahUKEwiBs4Sxs5DqAhXRlqQKHVOtDacQtwIwA3oECAAQAQ&url=https%3A%2F%2Fdocs.cypress.io%2Fguides%2Fgetting-started%2Ftesting-your-app.html&usg=AOvVaw3W0X5-tml7M4CzJogRtX1i) video and [this](https://github.com/sendilkumarn/generator-jhipster/tree/feat/add-cypress) branch  (this kind of follows the structure mentioned by Cypress team).